recipes/sail/all/conanfile.py

```diff
@@ -0,0 +1,132 @@
+from conan.tools.files import rename
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+class SAILConan(ConanFile):
+    name = "sail"
+    version = "0.9.0"
```

Comment:
```suggestion
```

Reply:
Can we have a recipe without a version, and set version after a release of sail?

Reply:
As far as I can see from the docs, we can have pre-release versions. Is it true? If yes, I can wait for the next pre-release and update this version here to `0.9.0-pre22`